lib/spack/docs/basic_usage.rst

```diff
@@ -2,7 +2,7 @@
 
 Basic usage
 =====================
-
+p
```

Comment:
Oh yeah, that's right.  Changes made in the GitHub UI leave me with a real fork of my own.  I'll fix it there.